themes/classic/assets/css/theme.css

```diff
@@ -4823,7 +4818,11 @@ ul {
     margin-top: 1rem; }
   section.checkout-step {
     width: 100%; }
-  section.checkout-step input[name=address1], section.checkout-step input[name=address2], section.checkout-step input[type=text], section.checkout-step input[type=password], input[type=date] {
+  section.checkout-step input[name=address1],
+  section.checkout-step input[name=address2],
+  section.checkout-step input[type=text],
+  section.checkout-step input[type=password],
```

Comment:
Nah, I didn't agree, this is a common thing - modyfing checkout fields - and for sure we're aiming to have it available within a module with proper hook, not an override. 

But ok, I didn't see alternative now because fields are in loop, without {in_array} it can be a little problem to style only some fields.